sympy/simplify/hyperexpand.py

```diff
@@ -2304,7 +2306,7 @@ def do_slater(an, bm, ap, bq, z, zfinal):
                 # Now sum the finitely many residues:
                 # XXX This speeds up some cases - is it a good idea?
                 integrand = expand_func(integrand)
-                for r in range(lu):
+                for r in range(int(lu)):
```

Comment:
I assumed int would take the closest integer. But if it rounds down, it should  definitely be round(lu).